apps/management/lib/management/view_state.ex

```diff
@@ -0,0 +1,62 @@
+defmodule Management.ViewState do
+  @moduledoc """
+  View state management behaviour and default implementation on a per-collection
+  basis. `ViewState` collections should track only one type of object.
+
+  ## Configuration
+
+  Configure `ViewState` with `use/2` macro.
```

Comment:
technically it's a `use/1` since it's looking for a keyword list.

Reply:
First argument, `Management.ViewState`. Second argument, configuration.

Reply:
good point; wrong context.